routes/benine.js

```diff
@@ -46,4 +46,67 @@ router.get("/recallTest", (req, res) => {
         });
 });
 
+router.post("/cameras/:cameraId(\\d+)/set-remote-camera-id", (req, res) => {
```

Comment:
I believe you switched the routes here. Also, could you add comments to the routes? What do they do?